services-all/pom.xml

```diff
@@ -27,7 +27,19 @@
                 <groupId>org.mule.services</groupId>
                 <artifactId>mule-service-http</artifactId>  
                 <version>${muleHttpServiceVersion}</version>
-            </dependency>  	
+            </dependency>
+            <dependency>
+                <groupId>org.mule.services</groupId>
+                <artifactId>mule-service-oauth</artifactId>
+                <version>${muleOAuthServiceVersion}</version>
```

Comment:
remove the versions from the dependencies in the `dependencies` section (http too now that you're at ti)